appcues/src/main/java/com/appcues/util/DoubleExt.kt

```diff
@@ -0,0 +1,7 @@
+package com.appcues.util
+
+import kotlin.math.abs
+
+private const val TOLERANCE = 0.000001
+internal infix fun Double.eq(other: Double) = abs(this - other) <= TOLERANCE
+internal infix fun Double.ne(other: Double) = abs(this - other) > TOLERANCE
```

Comment:
@andretortolano what do you think of this approach? added these two infix ops for equal and not equal double comparison with tolerance.  Used in `ModifierExt` and `SharedModal` above.

Reply:
thats cool

Reply:
i think `ne` is not a very clear name - maybe `noteq` ?